javaparser-core/src/main/java/com/github/javaparser/ast/body/FieldDeclaration.java

```diff
@@ -31,18 +38,12 @@
 import com.github.javaparser.ast.visitor.GenericVisitor;
 import com.github.javaparser.ast.visitor.VoidVisitor;
 
-import java.util.ArrayList;
-import java.util.List;
-
-import static com.github.javaparser.Position.pos;
-import static com.github.javaparser.ast.internal.Utils.*;
-
```

Comment:
Be careful with automatically organizing imports, it causes a lot of nonsense in the commits.